bears/setup.py.in

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python3
+
+import locale
+
+import setuptools.command.build_py
+from setuptools import find_packages, setup
+
+try:
+    locale.getlocale()
+except (ValueError, UnicodeError):
+    locale.setlocale(locale.LC_ALL, 'en_US.UTF-8')
+
+if __name__== "__main__":
+	setup(name='$NAME',
+		  version=$VERSION,
+		  description='The $NAME bear for coala (http://coala.rtfd.org/)',
```

Comment:
yeah i think this description here is supposed to be rather short and descriptive, maybe i can make a ``long_description`` parameter which reads the docstrings if this is what we want.